optimum/exporters/ipex/cache_utils.py

```diff
@@ -67,7 +67,7 @@ def __init__(
         self.max_cache_len = max_cache_len
         self.num_kv_heads = config.num_key_value_heads
         self.num_hidden_layers = config.num_hidden_layers
-        if hasattr(config, "head_dim"):
+        if getattr(config, "head_dim", None) is not None:
```

Comment:
why change this? I suppose they are equivalent.

Reply:
The config has the attr `head_dim` but the value is None
![Uploading image.png…]()